doozerlib/distgit.py

```diff
@@ -935,6 +941,17 @@ def _extract_container_build_logs(self, task_id):
         except OSError as e:
             self.logger.warning("Exception while trying to extract build logs in {}: {}".format(logs_dir, e))
 
+    def add_missing_pkgs(self, pkg_name):
+        """
+        add missing packages to runtime.missing_pkgs set and print it out for
+        `brew tag-build` from `rhaos-4.0-rhel-7-candidate` to `rhaos-4.1-rhel-7-candidate` manually
+        :param pkg_name: Missing packages like: No package (.*) available
+        """
+        latest = image.exec_latest_build_info("rhaos-4.0-rhel-7-candidate", pkg_name)
```

Comment:
tag cannot be hardcoded but i'm not sure where to get it :)

get from group.yml branch and decrement x in 4.x